bridge_adaptivity/module/templates/module/modals/congratulation_message_modal.html

```diff
@@ -0,0 +1,18 @@
+{% load bootstrap3 %}
+
+<!-- Modal -->
+<div class="modal fade" id="congratulationMessage" role="dialog">
+  <div class="modal-dialog" role="dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span
+          aria-hidden="true">&times;</span></button>
+          <h3 class="modal-title">Congratulations!</i></h3>
+      </div>
+      <div class="modal-body">
+          <h4>You have achieved an excellent score on this assignment. This is a good time to move on to the next part of the course.</h4>
```

Comment:
I would prefer we let the instructor update Congratulation text.

Reply:
I wrote the question about it ti Colin.